source/api/presentation/3.0/index.md

```diff
@@ -950,11 +955,11 @@ The value of the `@context` property _MUST_ be a list, and the __last__ two valu
 }
 ```
 
-Any additional fields beyond those defined in this specification or the Web Annotation Data Model _SHOULD_ be mapped to RDF predicates using further context documents.   If possible, these extensions _SHOULD_ be added to the top level `@context` field, and _MUST_ be added before the above contexts.  The JSON-LD 1.1 functionality of type and predicate specific context definitions _SHOULD_ be used if possible to try to minimize any cross-extension collisions.
+Any additional fields beyond those defined in this specification or the Web Annotation Data Model _SHOULD_ be mapped to RDF predicates using further context documents.   These extensions _SHOULD_ be added to the top level `@context` field, and _MUST_ be added before the above contexts.  The JSON-LD 1.1 functionality of [type and predicate specific context definitions][json-ld-scoped-contexts] _MUST_ be used to minimize cross-extension collisions.
```

Comment:
> The JSON-LD 1.1 functionality of [type and predicate specific context definitions][json-ld-scoped-contexts] _MUST_ be used to minimize cross-extension collisions.

Maybe refer to the term "scoped contexts" since this is the name of the section to which you are linking, and that terminology is used in the next para?